cybench/models/residual_models.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import pandas as pd
+import pickle
+
+from cybench.datasets.dataset import Dataset
+from cybench.datasets.modified_dataset import ModifiedTargetsDataset
+from cybench.models.model import BaseModel
+from cybench.models.trend_models import TrendModel
+from cybench.models.sklearn_models import SklearnRidge, SklearnRandomForest
+from cybench.models.nn_models import BaselineLSTM, BaselineInceptionTime
+from cybench.util.data import data_to_pandas
+
+from cybench.config import (
+    KEY_LOC,
+    KEY_YEAR,
+    KEY_TARGET,
+)
+
+
+class ResidualModel(BaseModel):
+    def __init__(self, baseline_model: BaseModel):
+        """Ridge model that predicts residuals from the trend.
+
+        It leverages `TrendModel` to estimate the trend.
+        Then it detrends targets and predicts the residuals.
+        During inference, residual predictions are added to trend to
+        produce the target predictions.
+        """
+        self._trend_model = TrendModel()
+        self._baseline_model = baseline_model
+
+    def fit(self, dataset: Dataset, **fit_params):
+        """Fit or train the model.
+
+        Args:
+          train_dataset (Dataset): training dataset
+          **fit_params: Additional parameters.
+
+        Returns:
```

Comment:
It seems it doesn't return anything (by design); adjust comment accordingly

Reply:
It should return actually.